cpu/cortexm_common/cortexm_init.c

```diff
@@ -26,6 +28,8 @@
  */
 #define FULL_FPU_ACCESS         (0x00f00000)
 
+extern const void *interrupt_vector[];
```

Comment:
`interrupt_vector` is defined in the CPU-specific `vectors.c`. The goal of this PR is not to override the vector table but to allow placement of it (and likely the entire RIOT image) somewhere other than `CPU_FLASH_BASE`. The easiest way to do so is to modify the `ORIGIN` of the `rom` segment in the CPU-specific linker script.